gravitee-policy-ratelimit/src/main/resources/schemas/schema-form.json

```diff
@@ -63,8 +63,18 @@
                 "periodTime": {
                     "type": "integer",
                     "title": "Time duration",
+                    "description": "Static value of time duration (This value is used when greater than 1).",
+                    "minimum": 1,
```

Comment:
I would have put a minimum at 0 and a default value to 0. Then you can use the dynamic configuration when the value of the static field is 0.
If this static value is strictly positive, then you ignore the dynamic configuration.

WDYT?